analyzer/src/funTest/assets/projects/synthetic/gradle-android-cyclic/app/src/main/res/drawable-v24/ic_launcher_foreground.xml

```diff
@@ -0,0 +1,34 @@
+<vector xmlns:aapt="http://schemas.android.com/aapt"
```

Comment:
Could we strip all resources to keep the number of files in this synthetic project small?

Reply:
Actually, I am not sure whether this test should be added. Currently it is failing (for reasons that need to be investigated); but if it runs through, it takes a rather long time (almost 20 minutes on my machine) - these Android projects are really heavy-weighted.

Unfortunately, I was unable to come up with a minimum synthetic test that reproduces the issue.

What is your opinion about the test?

Reply:
If it's really taking 20 minutes, then maybe we could still add it but tag it with `AndroidTag` and `ExpensiveTag`, and again start not running those in CI.

Reply:
I have added the mentioned tags. Let's see whether the test passes now, then I can promote this PR to 'ready to review'.